### PR TITLE
Issue 814 : Name the zip file after the folder name

### PR DESCRIPTION
**Pull request by fred1701d**
The name of the zip when downloading a folder should be named after the folder.
For root, default will be gallery.zip

### CODE REVIEW

File: src/backend/middlewares/GalleryMWs.ts [hunk 1]
```diff
@@ -84,6 +84,7 @@ export class GalleryMWs {
     res: Response,
     next: NextFunction
   ): Promise<void> {
+    var contentDisposition = require('content-disposition')
```
Comment (bpatrik) [inline src/backend/middlewares/GalleryMWs.ts:87]: please fix the lint errors.

File: src/backend/middlewares/GalleryMWs.ts [hunk 2]
```diff
@@ -102,7 +103,16 @@ export class GalleryMWs {
 
     try {
       res.set('Content-Type', 'application/zip');
-      res.set('Content-Disposition', 'attachment; filename=Gallery.zip');
+      var zipName = directoryName;
```
Comment (bpatrik) [inline src/backend/middlewares/GalleryMWs.ts:106]: please fix the lint errors.

File: src/backend/middlewares/GalleryMWs.ts (hunk 1 above)
Comment (bpatrik) [inline src/backend/middlewares/GalleryMWs.ts:87]: Is this a new dependency to the project? It is not listed in the package.json
Can you reason about why this is needed?

**Review by bpatrik — CHANGES_REQUESTED** [review]

File: src/backend/middlewares/GalleryMWs.ts (hunk 1 above)
Comment (fred1701d) [reply src/backend/middlewares/GalleryMWs.ts]: I don't know how to fix it. I can decline my PR
I'm not a Node JS developer. Java is much better :-)

Comment (fred1701d) [reply src/backend/middlewares/GalleryMWs.ts]: I don't know, that's something I found online.
I think I'll just drop my PR

**Pull request closed without merging**